cleanlab/benchmarking/noise_generation.py

```diff
@@ -116,7 +116,7 @@ def generate_noisy_labels(true_labels, noise_matrix):
     Parameters
     ----------
     true_labels : np.array
-      An array of shape ``(N,)`` representing perfect labels, without any
+      An array of shape ``(N, )`` representing perfect labels, without any
```

Comment:
this was convention at other places i've worked because it reduces edit distance if you need to add a second dimension. It's also clearer to have separation between two punctuation with different meaning.

Reply:
will change back to numpy convention to move things forward in agreement